src/views/mixed-content/mixed-content-by-type/index.js

```diff
@@ -0,0 +1,66 @@
+import Card from '@material-ui/core/Card';
+import {CardContent} from '@material-ui/core';
+import React, {useEffect, useState} from 'react';
+import {api} from '../../../client';
+import {ResponsivePie} from '@nivo/pie';
+import {ChartContainer} from '../../chart-container';
+import Typography from '@material-ui/core/Typography';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import {SelectContainer} from '../../select-container';
+
+/**
+ * This component shows a pie chart with the percentage of mixed
```

Comment:
As Livvie recommended [here](https://github.com/googleinterns/step180-2020/pull/2#discussion_r464121662), we can rephrase this into something like:

```
Returns a pie chart with the percentage of mixed content by types
```